spec/unit/pull_request_spec.rb

```diff
@@ -23,26 +74,14 @@
     let(:pull_request) { PullRequest.new data }
     let(:title) { Faker::Lorem.sentence }
 
-    subject { pull_request.to_commit[:source_commit] }
-
-    it 'wraps the whole thing in a source_commit key' do
-      expect(pull_request.to_commit.keys).to be == [:source_commit]
-    end
-
-    it 'exports the login name as the author' do
-      expect(subject[:author]).to be == login
-    end
-
-    it 'exports the number as the commit ID' do
-      expect(subject[:commit_id]).to be == "pulls/#{number}"
-    end
+    subject { pull_request.to_comment }
 
-    it 'exports the title in the message field, with prefix' do
-      expect(subject[:message]).to be == "Created pull request: #{title}"
-    end
-
-    it 'exports the URL' do
-      expect(subject[:url]).to be == html_url
+    it 'converts the pull request data to a string' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.